web/src/api/hooks/taxonomies.ts

```diff
@@ -18,12 +18,12 @@ import { useBadgerFetch } from './api';
 const namespace = process.env.REACT_APP_TAXONOMIES_API_NAMESPACE;
 
 export const useTaxonomies: QueryHookType<QueryHookParamsType<Taxon>, PagedResponse<Taxon>> = (
-    { page, size, searchText, filters, sortConfig },
+    { page, size, searchText, searchField, filters, sortConfig },
     options
 ) => {
     return useQuery(
         ['taxonomies', page, size, searchText, filters, sortConfig],
-        () => taxonsFetcher(page, size, searchText, filters, sortConfig),
+        () => taxonsFetcher(page, size, searchText, filters, sortConfig, searchField),
```

Comment:
do we need to add searchField if it's always 'name'?

Reply:
it can be `undefined`
<img width="290" alt="image" src="https://user-images.githubusercontent.com/50149163/210774882-bea79c79-b082-4868-b90a-9ca7e56d8b02.png">